README.md

```diff
@@ -77,19 +97,14 @@ Analyzing 175 revisions...
 ( 12; 175)              /src
 ```
 
-The ranges on the left indicate
-at what revision folders appeared first and got deleted latest.
-You can see a few things in this very output:
-
- * That a branch called "libbs2b-config-header" got deleted
-   at revision 76.
+The ranges on the left indicate at what revision folders appeared first and got deleted latest.
 
- * In which order bs2b plug-ins appeared over time.
+You can see a few things in this output:
+* That a branch called "libbs2b-config-header" got deleted at revision 76.
+* In which order plug-ins in `plugins` appeared over time.
```

Comment:
I wonder if we should use backticks for more values around here if we use it for `plugins`.